### PR TITLE
Add support for custom `task` location or command

### PR DESCRIPTION
**Pull request by robgolding**
Thanks @xarthurx for proposing WSL support in #62. This commit adds support in a different way, by allowing the "task" command to be customised (to `wsl task`, for example).

### CODE REVIEW

**Comment by xarthurx** [comment]
@robgolding is this PR done?
I'm a bit busy this week but will give it a run during weekends if it is done.

**Comment by robgolding** [comment]
The tests have failed here but only because `wsl` isn't present on CI—I'll need to figure out a way around that. It should be ready to give this a go locally @xarthurx!

**Comment by robgolding** [comment]
@xarthurx OK, I switched to using `TASKRC` env var. Can you give this another go?

File: tasklib/backends.py [hunk 1]
```diff
@@ -95,13 +95,15 @@ class TaskWarrior(Backend):
     VERSION_2_4_5 = six.u('2.4.5')
 
     def __init__(self, data_location=None, create=True,
-                 taskrc_location='~/.taskrc', task_command='task'):
-        self.taskrc_location = os.path.expanduser(taskrc_location)
+                 taskrc_location=None, task_command='task'):
```
Comment (xarthurx) [inline tasklib/backends.py]: why cancel the default `taskrc_location` while the original one works fine on windows?

Comment (xarthurx) [reply tasklib/backends.py]: Does this mean that users will always need to specify a `rc` file by themself?

**Comment by xarthurx** [comment]
@robgolding  This works fine! :)

**Comment by robgolding** [comment]
Great, thanks @xarthurx. In that case since CI is now passing I'll get this merged and release a new version. Thanks again for the help with this!

**Pull request merged**